src/js/components/chat-input.jsx

```diff
@@ -35,8 +36,8 @@ export class ChatInputComponent extends Component {
     }
 
     onFocus(e) {
-        if (getReadTimestamp() > 0) {
-            updateReadTimestamp(0);
+        if (store.getState().conversation.unreadCount > 0) {
```

Comment:
Yes, you can add the `conversation` state in the `connect` call at the end of the file I think